Website/pages/docs/development/Quickstart.mdx

```diff
@@ -1,11 +1,16 @@
+import { Callout } from "nextra/components";
+
 # Quick Start
 
 This is a quick start guide to get developers up and running with the project for development purposes.
 
 ## Clone the Project
 
-> [!NOTE]
-> You need to have Git installed on your machine to clone the project. If you don't have Git installed, you can download it from [here](https://git-scm.com/downloads).
+<Callout type="info">
+  You need to have Git installed on your machine to clone the project. If you
+  don't have Git installed, you can download it from
+  [here](https://git-scm.com/downloads).
+</Callout>
```

Comment:
@abhilashreddysh Does a **warning callout** suit here better? What do you think?